### DISCUSSION TITLE
Pseudo-classes and pseudo-elements are recurring problems

### DISCUSSION THREAD

**Discussion by virtualhasan** [tailwindlabs/tailwindcss#11503] [Ideas]
I think it is a boring process. Where pseudo has to be written repeatedly. This problem can be solved by following rule. Please consider this.

`<div class="bg-white h-32 w-32 dark:[bg-slate-500 text-lg text-red-400] before:[content-[''] h-full w-full bg-green-500]"></div>`

dark:[bg-red-500 flex justify-center items-center p-4 m-5]
hover:[border-1 flex justify-center items-center p-4 m-5 rounded]

**Comment by wongjn** [comment]
There have been many suggestions about adding such a feature. The maintainers have addressed this request before, that you may wish to read up on if you have not already done so:
- [A Twitter thread from a maintainer, Adam Wathan](https://twitter.com/adamwathan/status/1461519820411789314)
- [A previous GitHub discussion thread with maintainer responses](https://github.com/tailwindlabs/tailwindcss/discussions/8337#discussioncomment-4032611)
